app/code/Magento/MediaGalleryUi/etc/adminhtml/system.xml

```diff
@@ -9,9 +9,9 @@
     <system>
         <section id="system">
             <group id="media_gallery" translate="label" type="text" sortOrder="1000" showInDefault="1" showInWebsite="0" showInStore="0">
-                <label>Enhanced Media Gallery</label>
+                <label>Media Gallery</label>
                 <field id="enabled" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
-                    <label>Enabled</label>
+                    <label>Enable Old Media Gallery</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
```

Comment:
@konarshankar07 can you please keep the existing value for enabled media gallery and just change the option values for Stores Configuration.